eq-author-api/constants/googleServiceKeys.js

```diff
@@ -0,0 +1,24 @@
+const logger = require("pino")();
+
+module.exports.VERIFY_PUBLIC_KEYS_URL =
+  "https://www.googleapis.com/robot/v1/metadata/x509/securetoken@system.gserviceaccount.com";
+
+const JwtAlgorithms = ["RS256"];
+const JwtTokenIssUrl = "https://securetoken.google.com";
+
+if (!process.env.REACT_APP_FIREBASE_PROJECT_ID) {
```

Comment:
Can we rename to drop the `REACT_APP`? I think we only use environment variables prefixed with this on the front-end.

Reply:
Done